test/e2e/autoscale_hpa_test.go

```diff
@@ -139,6 +150,25 @@ func assertHPAAutoscaleUpToNumPods(ctx *TestContext, targetPods float64, done <-
 	}
 }
 
+func assertMemoryHPAAutoscaleUpToNumPods(ctx *TestContext, targetPods float64, done <-chan time.Time, quick bool) {
```

Comment:
if we had a generic `generateTrafficAtFixedConcurrencyWithLoad` method, then we could have a single `assertHPAAutoscaleUpToNumPods` that accepted `vegetaParam`/`vegetaValue` and we could call it as `assertHPAAutoscaleUpToNumPods(ctx, 100, "bloat", bloat, done, true)`, rather than needing to duplicate the method (mostly because the method name is getting a little unwieldy here I think).